isolate_example/lib/page_two.dart

```diff
@@ -40,14 +40,12 @@ class InfiniteProcessPage extends StatelessWidget {
             children: [
               Row(
                 mainAxisAlignment: MainAxisAlignment.center,
-                children: [
-                  newButtons(context),
-                ],
+                children: [newButtons(context)],
               ),
               Row(
                 mainAxisAlignment: MainAxisAlignment.center,
                 crossAxisAlignment: CrossAxisAlignment.center,
-                children: <Widget>[
+                children: [
                   Switch(
                     value:
                         !(Provider.of<IsolateController>(context, listen: true)
```

Comment:
This code would be tidier if there was a single call to get the provider at the top of the build method. Something like:

```
final controller = Provider.of<IsolateController>(context, listen: true);
```

Then you can set `value` and `onChanged` here using `!controller.paused` and `controller.pausedSwitch()`